sdk/communication/azure-communication-chat/src/samples/java/com/azure/communication/chat/CreateChatThread.java

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.chat;
+
+import com.azure.communication.chat.models.ChatParticipant;
+import com.azure.communication.chat.models.CreateChatThreadOptions;
+import com.azure.communication.chat.models.CreateChatThreadResult;
+import com.azure.communication.chat.models.SendChatMessageOptions;
+import com.azure.communication.chat.models.SendChatMessageResult;
+import com.azure.communication.common.CommunicationTokenCredential;
+import com.azure.communication.common.CommunicationUserIdentifier;
+import com.azure.core.http.rest.PagedIterable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CreateChatThread {
+    public static void main(String[] args) {
+        String endpoint = System.getenv("AZURE_COMMUNICATION_ENDPOINT");
```

Comment:
With samples, we usually put a fake value. E.g.
```
String endpoint = "https://<RESOURCE_NAME>.communication.azure.com";
 String accessKey = "SECRET";
```

Reply:
to be in sync with other sdks we what to make this a runable example

Reply:
See comment above